src/util/configuration.h

```diff
@@ -1,40 +1,85 @@
-
 #ifndef UTIL_CONFIGURATION_H_
 #define UTIL_CONFIGURATION_H_
 
 #include <string>
 #include <map>
+#include "cpptoml.h"
```

Comment:
This doesn't work for me:
> ../mapping-core/src/util/configuration.h:6:19: fatal error: cpptoml: No such file or directory
 #include "cpptoml"

Reply:
Are you maybe missing the` .h` in the code? I can get the exact same error message when I remove the `.h` from `#include "cpptoml.h"` but otherwise the message always states not finding `cpptoml.h` (if I remove cpptoml from CMake).
Generally I tried deleting the files CMake downloaded for cpptoml to reproduce this, but I can't at the moment.

Reply:
You are right, it works for mapping_core

However, when compiling together with a mapping mode, e.g. mapping_r, it breaks. For this, check out mapping_r to the same directory as mapping_core and add the following CMake-options:
`-DMAPPING_MODULES=mapping-r`
It cannot find the cpptoml.h then. Can you please test whether it works for you?